### PR TITLE
add diversity statement and rspec test

### PR DESCRIPTION
**Pull request by bri616**
I added one small change in the rspec tests, as well, one of the descriptions had the word "speakers" when it should have said "schedule"

let me know if I can change anything in the styling or whatnot of the statement.

### CODE REVIEW

**Comment by kmanion** [comment]
I read over it, and :+1:

File: app/views/diversity.markdown [hunk 1]
```diff
@@ -0,0 +1,26 @@
...
+We think accessibility for people with disabilities is a priority, not an afterthought. We think neurodiversity is a feature, not a bug. We believe in being inclusive, welcoming, and supportive of anyone who comes to us with good faith and the desire to build a community.
+
+We have enough experience to know that we won't get any of this perfect on the first try. But we have enough hope, energy, and idealism to want to learn things we don't know now. We may not be able to satisfy everyone, but we can certainly work to avoid harming anyone. And we promise that if we get it wrong, we'll listen carefully and respectfully to you when you point it out to us, and we'll do our best to make good on our mistakes.
+We value empathy more than we value our code quality and our product quality. Not just empathy for the users of the things we make but for everyone who contributes as well. When any of us make a mistake, we are ready and willing to take the responsibility to correct it. If someone is hurt by us, it is always our first response to put ourselves in their shoes.
```
Comment (khjrtbrg) [inline app/views/diversity.markdown]: :grey_question: @bri616 Just curious, did you want this line and [line 15](https://github.com/OSFeels/OSFeels.com/pull/85/files#diff-5ba0150d775a358705540647f41a0ce0R15) to be separate paragraphs? It looks like they're getting smushed into the line above them when they're rendered.

Comment (bri616) [inline app/views/diversity.markdown]: Good point, yes-they should be separate - fixed it, I'll push it. :)

**Comment by khjrtbrg** [comment]
:sparkling_heart: Awesome, merging!

**Pull request merged**